pkg/remote/builtin/github/github.go

```diff
@@ -34,12 +37,14 @@ type GitHub struct {
 
 func New(conf *config.Config) *GitHub {
```

Comment:
>we're actually getting rid of the config file all together and moving to a driver-based model.

I see, thank you.
So, would the return type of `remote.Load(os.Getenv("REMOTE_DRIVER)) `  be `Remote`?

I will see remote.
And I'm looking forward to get GitLab and Gogs driver models.